.circleci/config.yml

```diff
@@ -1,6 +1,26 @@
 version: 2
 
+apt-run: &apt-install
+  name: Install apt packages
+  command: |
+    sudo apt update
+    sudo apt install -y graphviz
+
+pip-run: &pip-install
+  name: Install Python dependencies
+  command: |
+    python3 -m venv venv
+    . venv/bin/activate
+    pip install -q -r pip-requirements-doc
```

Comment:
Do these really have to be references since they are only used once?

Reply:
Oh see I find it much nicer to have them like this. I find it seperates out the different stages better and makes the interesting bit more obvious.